core/src/main/java/hudson/model/Descriptor.java

```diff
@@ -474,6 +474,12 @@ public void calcAutoCompleteSettings(String field, Map<String, Object> attribute
         if (method == null)
             return;    // no auto-completion
 
+        // build query parameter line by figuring out what should be submitted
+        List<String> depends = buildFillDependencies(method, new ArrayList<>());
+        if (!depends.isEmpty()) {
+            attributes.put("fillDependsOn", String.join(" ", depends));
```

Comment:
I thought about whether this should be a different attribute name, e.g. `autocompleteFillDependsOn` but I couldn't see when they would ever clash so I just left it as the same name